src/app/components/list/overview/index.html

```diff
@@ -75,6 +75,180 @@
+      isOptional="true"
+    >
+    Specifies a function to dynamically manage the data source when users change the text in the list field. The search function must return an array or a promise of an array. The <sky-code>search</sky-code> property is particularly useful when the data source does not live in the source code. This property accepts <a stacheRouterLink="." fragment="listsearchmodel-properties">a <sky-code>ListSearchModel</sky-code> value</a>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="sort"
+      isOptional="true"
+    >
+    Specifies a set of fields to sort by. This property accepts <a stacheRouterLink="." fragment="listsortmodel-properties"> <sky-code>ListSortModel</sky-code> value</a>.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListFilterModel properties">
+    <sky-demo-page-property
+      propertyName="defaultValue"
+      isOptional="true"
+    >
+    Specifies the default value of the filter. When the filter is set to the default value, it is not applied. This property accepts <sky-code>any</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="dismissible"
+      isOptional="true"
+    >
+    Indicates whether users can dismiss the filter's summary item. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="filterFunction"
+      isOptional="true"
+    >
+    Specifies a function to determine whether items are filtered. This property is required when using an in-memory data provider. This property accepts a function of type <sky-code>(item: ListItemModel, filter: any) => boolean</sky-code>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="label"
+      isOptional="true"
+    >
+    Specifies a label for the filter's summary item. The label defaults to the <sky-code>value</sky-code> of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="name"
+      isOptional="true"
+    >
+    Specifies the name of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="value"
+      isOptional="true"
+    >
+    Specifies the current value of the filter. This property accepts <sky-code>any</sky-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSearchModel properties">
+    <sky-demo-page-property
+      propertyName="fieldSelectors"
+      isOptional="true"
+    >
+    Specifies the columns to search. The columns correspond to <sky-code>field</sky-code> values that you specify with the <sky-code>data</sky-code> property. This property accepts an array of <sky-code>string</sky-code> values. 
```

Comment:
Do you think we should specify where the `data` property comes from? Either by calling it out (i.e., "...with the list component's `data` property") or linking to that section (i.e., making "the `data` property" a link to the section above)?

Reply:
Updating this with the first suggested tweak. Feel free to undo this change if you deliberately left this as is.